From 8de179781269345fe2621b21951173380f331830 Mon Sep 17 00:00:00 2001
From: Markus Armbruster <armbru@redhat.com>
Date: Wed, 31 Mar 2010 13:25:06 -0300
Subject: [PATCH 51/66] error: New QERR_DEVICE_INIT_FAILED

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1270041921-28969-52-git-send-email-armbru@redhat.com>
Patchwork-id: 8219
O-Subject: [PATCH 51/66] error: New QERR_DEVICE_INIT_FAILED
Bugzilla: 579470
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Luiz Capitulino <lcapitulino@redhat.com>

From: Markus Armbruster <armbru@pond.sub.org>

(cherry picked from commit 4d9a1a15214c01dcbff2d7c02cef027e80442d79)

Conflicts:

	qerror.c
	qerror.h
---
 qerror.c |    4 ++++
 qerror.h |    3 +++
 2 files changed, 7 insertions(+), 0 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 qerror.c |    4 ++++
 qerror.h |    3 +++
 2 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/qerror.c b/qerror.c
index 6dd5eab..512a356 100644
--- a/qerror.c
+++ b/qerror.c
@@ -65,6 +65,10 @@ static const QErrorStringTable qerror_table[] = {
         .desc      = "Device '%(device)' is not encrypted",
     },
     {
+        .error_fmt = QERR_DEVICE_INIT_FAILED,
+        .desc      = "Device '%(device)' could not be initialized",
+    },
+    {
         .error_fmt = QERR_DEVICE_LOCKED,
         .desc      = "Device '%(device)' is locked",
     },
diff --git a/qerror.h b/qerror.h
index 4e68c08..e0ce99a 100644
--- a/qerror.h
+++ b/qerror.h
@@ -60,6 +60,9 @@ QError *qobject_to_qerror(const QObject *obj);
 #define QERR_DEVICE_NOT_ENCRYPTED \
     "{ 'class': 'DeviceNotEncrypted', 'data': { 'device': %s } }"
 
+#define QERR_DEVICE_INIT_FAILED \
+    "{ 'class': 'DeviceInitFailed', 'data': { 'device': %s } }"
+
 #define QERR_DEVICE_LOCKED \
     "{ 'class': 'DeviceLocked', 'data': { 'device': %s } }"
 
-- 
1.7.0.3

